### PR TITLE
Upgrade eslint 

### PR DESCRIPTION
**Pull request by MaxBittker**
have other stuff to do, so thought this would be easiest /shrug

### CODE REVIEW

File: src/sentry/static/sentry/app/components/fileChange.jsx [hunk 1]
```diff
@@ -26,8 +26,8 @@ const FileChange = React.createClass({
   },
 
   render() {
-    let {filename, authors, types} = this.props;
-    types = Array.from(types);
+    let {filename, authors} = this.props;
+    // types = Array.from(types);
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/fileChange.jsx]: Is this variable unused? Why not just delete it?

File: src/sentry/static/sentry/app/utils.jsx [hunk 2]
```diff
@@ -23,9 +23,7 @@ export const arrayIsEqual = function(arr, other, deep) {
     return false;
   }
 
-  for (let i = 0, l = Math.max(arr.length, other.length); i < l; i++) {
-    return valueIsEqual(arr[i], other[i], deep);
-  }
+  return arr.every((val, idx) => valueIsEqual(val, other[idx], deep));
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/utils.jsx]: Ouch – uh, so was this completely busted? And if so, what effect will this have elsewhere?

File: src/sentry/static/sentry/app/views/groupActivity/index.jsx [hunk 3]
```diff
@@ -99,10 +99,6 @@ const GroupActivity = React.createClass({
             return t('%s assigned this event to an unknown user', author);
           }
         }
-        return t('%(author)s assigned this event to %(assignee)s', {
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/groupActivity/index.jsx]: If I'm reading this correctly – never hits?

File: src/sentry/static/sentry/app/views/stream/actions.jsx [hunk 4]
```diff
@@ -156,13 +156,13 @@ const StreamActions = React.createClass({
...
+        ? (<div>
             <p>{t('This will apply to the current search query:')}</p>
             <pre>{this.props.query}</pre>
-          </div>
-        : <p className="error">
+          </div>)
+        : (<p className="error">
             <strong>{t('This will apply to ALL issues in this project!')}</strong>
-          </p>;
+          </p>);
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/stream/actions.jsx]: Hmm, what does Prettier do here?

File: src/sentry/static/sentry/app/views/teamMembers.jsx [hunk 5]
```diff
@@ -25,8 +25,7 @@ const TeamMembers = React.createClass({
   componentWillReceiveProps(nextProps) {
     let params = this.props.params;
     if (
-      nextProps.params.teamId !== params.teamId ||
-      nextProps.params.orgId !== params.orgId
+      nextProps.params.teamId !== params.teamId || nextProps.params.orgId !== params.orgId
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/teamMembers.jsx]: Why this change? Affected by Prettier?

**Review by benvinegar — CHANGES_REQUESTED** [review]
See comments

File: src/sentry/static/sentry/app/utils.jsx (hunk 2 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/utils.jsx]: this will eliminate some corner case false positive matches. afaik it's not used too extensively in the app

File: src/sentry/static/sentry/app/components/fileChange.jsx (hunk 1 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/fileChange.jsx]: there's other commented out stuff in that file and i didn't want to step on any toes, it's recent release stuff

File: src/sentry/static/sentry/app/views/groupActivity/index.jsx (hunk 3 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/groupActivity/index.jsx]: yep, which eslint noticed

File: src/sentry/static/sentry/app/views/stream/actions.jsx (hunk 4 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/stream/actions.jsx]: weird, i don't know what happened here. when i ran prettier via my editor it went away

File: src/sentry/static/sentry/app/views/teamMembers.jsx (hunk 5 above)
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/teamMembers.jsx]: yeah, not sure why now

Comment (benvinegar) [inline src/sentry/static/sentry/app/views/teamMembers.jsx]: This looks like you ran an old version of Prettier, because 1.2.2 handles this case but 1.0.0 didn't.

**Review by benvinegar — APPROVED** [review]
🥇

File: yarn.lock [hunk 6]
```diff
@@ -4261,7 +4119,7 @@ preserve@^0.2.0:
   version "0.2.0"
   resolved "https://registry.yarnpkg.com/preserve/-/preserve-0.2.0.tgz#815ed1f6ebc65926f865b310c0713bcb3315ce4b"
 
-prettier@^1.2.2:
+prettier@1.2.2:
```
Comment (MaxBittker) [inline yarn.lock]: @benvinegar  is this ok?

Comment (benvinegar) [inline yarn.lock]: Yes because I changed it to 1.2.2 in package.json – but good catch
